### PR TITLE
chore: add contrinbution guidelines

### PR DESCRIPTION
**Pull request by Kenshiin13**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Contribution Guidelines
+
+If you're planning to contribute, **please follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard.**
+
+Pull Requests (PRs) that do not adhere to this standard will **not be accepted**.
+
+## Branch Requirements
+
+- **All PRs must be based on the `dev` branch.**
+- Merges will only occur into the `dev` branch before being released to the main branch.
```
Comment (Arctos2win) [inline CONTRIBUTING.md:10]: ...

Comment (Kenshiin13) [reply CONTRIBUTING.md:10]: I am above the rules

Comment (Arctos2win) [reply CONTRIBUTING.md:10]: no you are not :eyes: 
in the dev branch with this 💩

Comment (Kenshiin13) [reply CONTRIBUTING.md:10]: It's a draft for a reason my guy

Comment (Arctos2win) [reply CONTRIBUTING.md:10]: good boy

**Review by feelfreetofee — APPROVED** [review]

**Pull request merged**